graphs/omen-thumbnailmapping/src/schema.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  Entity,
+  Value,
+  ValueKind,
+  store,
+  Bytes,
+} from "@graphprotocol/graph-ts";
+
+export class OmenThumbnailMapping extends Entity {
+  constructor(marketAddress: Bytes) {
+    super();
+    this.set("marketAddress", Value.fromBytes(marketAddress));
+  }
+
+  save(): void {
+    let marketAddress = this.get("marketAddress");
+    let image_hash = this.get("image_hash");
+    assert(marketAddress != null, "Cannot save OmenThumbnailMapping entity without an marketAddress");
+    assert(image_hash != null, "Cannot save OmenThumbnailMapping entity without an image_hash");
+    if (marketAddress && image_hash) {
```

Comment:
You are already checking for this in the above

Reply:
True. I'm a typescript noob, but even with these asserts above, it complains that these variables can be null without the if statement:

![Screenshot by Dropbox Capture](https://github.com/gnosis/labs-contracts/assets/15619339/633b33ff-d271-4c62-ab71-78f6c20b1196)

This part of the code was actually mostly auto-generated as well, so I just left it as it was (with some minor changes). 

Do you have suggestions on how to do it differently?

Reply:
Ow, I didn't know this was auto-gen`ed. Then it's fine, it will be overwritten later nevertheless.
Let's keep this as is.